### ISSUE TITLE
The latest stable 1.9.x version is back to the Future :)

### ISSUE THREAD

**Issue by diev** [gsscoder/commandline#228]
```
#region License
// <copyright file="..." company="Giacomo Stelluti Scala">
//   Copyright 2015-2013 Giacomo Stelluti Scala
// </copyright>
```

20 **1** 5 - 2013, for all files

**Comment by gsscoder** [comment]
:smile: where did you find it?

**Comment by gsscoder** [comment]
Ah thanks, now I see... This is why I reduced the header at one line of comment or better be _beaten by Biff_!

The current master **2.0.*-beta** will be as soon promoted as stable: until that moment (or maybe permanently) I'll not commit or accept PR on previous stable.
(To be honest I'm accepting only fix also to master untile 2.0 -> stabilizes "100%").

We're a small team and the new version (now really usable) can do everything and more the old can does (w/ _terser syntax and functional core at its heart_), so just one "branch for time" will be supported.

Anyway I appreciated sharp eyes! :joy:
